src/Dmishh/Bundle/SettingsBundle/Controller/SettingsController.php

```diff
@@ -25,7 +25,14 @@ class SettingsController extends Controller
     public function manageGlobalAction(Request $request)
     {
         $securitySettings = $this->container->getParameter('settings_manager.security');
-        if (!empty($securitySettings['manage_global_settings_role']) && !$this->get('security.context')->isGranted(
+
+        if ($this->has('security.authorization_checker')) {
```

Comment:
This block should be extracted to a private method, which will be also used in `manageOwnAction`